docs/support.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Support
+---
+
+# Support
+
+## The Google HTTP Client Library for Java community
+
+### Ask development questions
+
+Ask  questions on StackOverflow:
+
+* Use the [google-api][so-google-api] tag, and optionally [java][so-java] or [android][so-android]
+  or [java google-app-engine][go-java-gae].
+* You can also use the [google-http-java-client][so-http-client] tag.
```

Comment:
go/so

Reply:
This one is correct. `so` is for Stack Overflow